### PR TITLE
Fix: Correct sending an undefined answer.

### PR DESCRIPTION
**Pull request by bjones1**
If a students clicks "check me" on an mchoice without selecting an answer, the code returns "undefined" instead of an empty string.

### CODE REVIEW

File: runestone/mchoice/js/mchoice.js [hunk 1]
```diff
@@ -533,7 +534,8 @@ export default class MultipleChoice extends RunestoneBase {
     }
 
     async logMCMFsubmission(sid) {
-        var answer = this.givenArray[0];
+        // If there's no answer provided (the array is empty), use a blank for the answer.
+        var answer = this.givenArray[0] || "";
```
Comment (bnmnetp) [inline runestone/mchoice/js/mchoice.js:538]: this seems like a good fix.

File: runestone/mchoice/js/mchoice.js [hunk 2]
```diff
@@ -394,8 +394,9 @@ export default class MultipleChoice extends RunestoneBase {
             if (buttonObjs[i].checked) {
                 given = buttonObjs[i].value;
                 this.givenArray.push(given);
-                this.feedbackString += `<li value="${i + 1}">${this.feedbackList[i]
-                    }</li>`;
+                this.feedbackString += `<li value="${i + 1}">${
+                    this.feedbackList[i]
+                }</li>`;
```
Comment (bnmnetp) [inline runestone/mchoice/js/mchoice.js:399]: This block looks like it should be hand formatted.  Not sure what formatter thinks it is a good idea to do `${` on one line and `}` after it makes it really hard to see that it is jQuery.

**Review by bnmnetp — APPROVED** [review]

File: runestone/mchoice/js/mchoice.js (hunk 2 above)
Comment (bjones1) [reply runestone/mchoice/js/mchoice.js:399]: I'll revert this. I use [Prettier](https://prettier.io/) to format JS.

Comment (bjones1) [reply runestone/mchoice/js/mchoice.js:399]: But also, this isn't JQuery, it's a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).

Comment (bjones1) [reply runestone/mchoice/js/mchoice.js:399]: Here's the code before Prettier:

``` JavaScript
                this.feedbackString += `<li value="${i + 1}">${this.feedbackList[i]
                    }</li>`;
```

This seems worse than what Prettier did to it!

Comment (bjones1) [reply runestone/mchoice/js/mchoice.js:399]: So on second thought, let's leave this change in.

File: runestone/mchoice/js/mchoice.js (hunk 1 above)
Comment (bjones1) [reply runestone/mchoice/js/mchoice.js:538]: Thanks!

**Pull request merged**